SendUserDataProcessingEmailActivity/index.ts

```diff
@@ -0,0 +1,51 @@
+﻿import * as NodeMailer from "nodemailer";
+
+import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
+
+import { MailUpTransport } from "io-functions-commons/dist/src/utils/mailup";
+
+import { getSendUserDataProcessingEmailActivityHandler } from "./handler";
+
+// Whether we're in a production environment
+const isProduction = process.env.NODE_ENV === "production";
+
+// Mailup
+const mailupUsername = getRequiredStringEnv("MAILUP_USERNAME");
+const mailupSecret = getRequiredStringEnv("MAILUP_SECRET");
+
+// Email data
+const EMAIL_TITLE = "IO - Richiesta di Download/Cancellazione Dati Utente";
```

Comment:
the email title is defined here while the text inside the activity. can we define both inside the activity function ? (so, just in case, we can put some user's data into the title as well)